com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/AdvancedOptionsUIBlock.cs

```diff
@@ -9,23 +9,36 @@
 
 namespace UnityEditor.Rendering.HighDefinition
 {
-    class AdvancedOptionsUIBlock : MaterialUIBlock
+    /// <summary>
+    /// Advanced options material UI block. 
+    /// </summary>
+    public class AdvancedOptionsUIBlock : MaterialUIBlock
     {
+        /// <summary>Advanced options features controls the visibility of fields in the block.</summary>
         [Flags]
         public enum Features
         {
+            /// <summary>Don't display any fields.</summary>
```

Comment:
Can you look at the casing in the summaries? They should reflect what the user sees in the UI

Reply:
When you set the feature to None, the block is displayed but empty (all the fields are hidden).
Would "Hide all the fields in the block." be okay for the doc?

Reply:
Yeah that sounds good